### DISCUSSION TITLE
viewing logs

### DISCUSSION THREAD

**Discussion by ondiekisteven** [adnanh/webhook#569] [Q&A]
How can i view logs generated by the server, or see how the webhook ran. I have setup webhooks.service which runs the server that receives the webhooks. Now i try to push code and see if the webhook will run but there's no response. How can i view the logs

**Comment by adnanh** [comment]
Not quite sure, but I think the logs should be sent to syslog if you've set it up as systemd service? Try running `journalctl -xfe` and then invoking webhook, and see if anything shows up.

**Comment by cxonexus-github-admin** [comment]
journalctl -u webhook -f will tail the service output

you can also specify -logfile on the command line to point to a specific output

When I installed via apt on ubuntu 20, it brought in an older version of webhook, so make sure you get the latest release from here and install it, or some of the newer options won't work.

**Comment by nicbou** [comment]
What about the command's log? I see logs from the webhook service, but not the command's output.
